samples/core/GetStarted/AspNetCore/EFGetStarted.AspNetCore.NewDb/EFGetStarted.AspNetCore.NewDb.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
Was this file missing or is it from a different PR?

Reply:
An oversight from a different PR, I'll delete it.  In the ASP.NET repo we have a .gitignore rule that excludes .sln files by default, so I get in the habit of not keeping track of when VS creates them, but when it does so in the EF repo there's no ignore rule and they get picked up.